src/overlays/actors/ovl_Demo_Gt/z_demo_gt.h

```diff
@@ -6,9 +6,22 @@
 
 struct DemoGt;
 
+typedef void (*DemoGtUpdateFunc)(struct DemoGt*, GlobalContext*);
+typedef void (*DemoGtDrawFunc)(struct DemoGt*, GlobalContext*);
+
 typedef struct DemoGt {
-    /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x5C];
+    /* 0x0000 */ DynaPolyActor dyna;
+    /* 0x0164 */ s32 updateMode;
+    /* 0x0168 */ s32 drawConfig;
+    /* 0x016C */ Vec3s unk_16C;
+    /* 0x0172 */ s16 unk_172;
+    /* 0x0174 */ s16 unk_174;
+    /* 0x0178 */ Vec3i unk_178;
+    /* 0x0184 */ char unk_184[0x4];
+    /* 0x0188 */ Vec3i unk_188;
+    /* 0x0194 */ char unk_194[0x4];
+    /* 0x0198 */ Vec3i unk_198;
+    /* 0x01A4 */ char unk_1A4[0x4];
```

Comment:
Considering their usage, these are probably just `s32[4]`
```suggestion
    /* 0x0178 */ s32 unk_178[4];
    /* 0x0188 */ s32 unk_188[4];
    /* 0x0198 */ s32 unk_198[4];
```